Userland/Libraries/LibGUI/Shortcut.h

```diff
@@ -49,7 +83,12 @@ template<>
 struct Traits<GUI::Shortcut> : public GenericTraits<GUI::Shortcut> {
     static unsigned hash(const GUI::Shortcut& shortcut)
     {
-        return pair_int_hash(shortcut.modifiers(), shortcut.key());
+        const unsigned base_hash = pair_int_hash(shortcut.modifiers(), (u32)shortcut.type());
```

Comment:
Is there a reason not to use `auto`? 
```suggestion
        auto base_hash = pair_int_hash(shortcut.modifiers(), (u32)shortcut.type());
```

Reply:
Nope! Just used to a non-auto C++ codebase haha